tools/data_integrity/score_instrument.php

```diff
@@ -0,0 +1,225 @@
+<?php
+/**
+ * This tools scores any registered instrument that was build using the
+ * NDB_BVL_Instrument class and that has a working score() method.
+ * The cmd-line arguments need to contain a valid test_name, 'all' or 'one' and a
+ * CandID and SessionID. 'all' scores all records of an instrument that belong to an
+ * active timepoint and whose data entry is completed and `administration` not equal to  'None'.
```

Comment:
to an active timepoint **with** data entry **that has been completed** and **administration**...